FlowCrypt/Controllers/Compose/ComposeViewController.swift

```diff
@@ -879,6 +888,14 @@ extension ComposeViewController {
         }
     }
 
+    private func isTextFieldValidEmail(textField: UITextField) -> Bool {
```

Comment:
Let's rename this to `showAlertIfTextFieldNotValidEmail`. It can still return `isValid` which will be made obvious with the `if let isValid`.

It's better to track the behavior of methods in their names, even if that makes for some weird names: fewer surprises.

Reply:
👍